static/app/views/performance/newTraceDetails/traceDrawer/tabs/nodeDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+import type {Location} from 'history';
+
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types';
+
+import {
+  isMissingInstrumentationNode,
+  isParentAutogroupedNode,
+  isSiblingAutogroupedNode,
+  isSpanNode,
+  isTraceErrorNode,
+  isTransactionNode,
+} from '../../guards';
+import type {TraceTree, TraceTreeNode} from '../../traceTree';
+import ErrorNodeDetails from '../details/errorNodeDetails';
+import MissingInstrumentationNodeDetails from '../details/missingInstrumentaionNodeDetails';
+import ParentAutogroupNodeDetails from '../details/parentAutogroupNodeDetails';
+import SiblingAutogroupNodeDetails from '../details/siblingAutogroupNodeDetails';
+import SpanNodeDetails from '../details/spanNodeDetails';
+import TransactionNodeDetails from '../details/transactionNodeDetails';
+
+export default function NodeDetail({
+  node,
+  organization,
+  location,
+}: {
+  location: Location;
+  node: TraceTreeNode<TraceTree.NodeValue> | null;
+  organization: Organization;
+}) {
+  if (!node) {
+    return <NoDetail>{t('Click on a row in the trace view for its details')}</NoDetail>;
```

Comment:
This should not be possible - if a user clicks away, then tab needs to switch to the trace tab as opposed to displaying an empty view